ironic-overcloud.yaml

```diff
@@ -14,9 +14,10 @@ parameter_defaults:
   IronicInspectorInterface: 'inspector'
 
 resource_registry:
-  OS::TripleO::Services::IronicApi: tripleo-heat-templates/deployment/ironic/ironic-api-container-puppet.yaml
-  OS::TripleO::Services::IronicConductor: tripleo-heat-templates/deployment/ironic/ironic-conductor-container-puppet.yaml
-  OS::TripleO::Services::IronicPxe: tripleo-heat-templates/deployment/ironic/ironic-pxe-container-puppet.yaml
-  OS::TripleO::Services::NovaIronic: tripleo-heat-templates/deployment/nova/nova-ironic-container-puppet.yaml
-  OS::TripleO::Services::IronicNeutronAgent: tripleo-heat-templates/deployment/ironic/ironic-neutron-agent-container-puppet.yaml
-  OS::TripleO::Services::NeutronDhcpAgent: tripleo-heat-templates/deployment/neutron/neutron-dhcp-container-puppet.yaml
+  OS::TripleO::Services::IronicApi: /usr/share/openstack-tripleo-heat-templates/deployment/ironic/ironic-api-container-puppet.yaml
+  OS::TripleO::Services::IronicConductor: /usr/share/openstack-tripleo-heat-templates/deployment/ironic/ironic-conductor-container-puppet.yaml
+  OS::TripleO::Services::IronicPxe: /usr/share/openstack-tripleo-heat-templates/deployment/ironic/ironic-pxe-container-puppet.yaml
+  OS::TripleO::Services::IronicInspector: /usr/share/openstack-tripleo-heat-templates/deployment/ironic/ironic-inspector-container-puppet.yaml
```

Comment:
We don't actually need IronicInspector if we set the IronicInspector interface, I think. I guess it doesn't hurt though!